raiden-ts/src/raiden.ts

```diff
@@ -411,8 +411,13 @@ export class Raiden {
       'raiden-ts': Raiden.version,
       'raiden-contracts': Raiden.contractVersion,
     });
-    // on complete, sets epicMiddleware to null, so this.started === false
-    this.deps.latest$.subscribe(undefined, undefined, () => (this.epicMiddleware = null));
+
+    // Set `epicMiddleware` to `null`, this indicates the instance is not running.
+    const observer_complete = {
```

Comment:
Please, use `camelCase`.. we only allow `snake_case` when dealing with external API schemas, local variables should always be camel, by typescript recommended eslint rules.